content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -11,7 +11,80 @@ aliases:
   - /influxdb/cloud/reference/release-notes/influxdb-cloud
 ---
 
-InfluxDB Cloud updates occur frequently. Find a compilation of monthly updates below.
+InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
+
+## January 2021
+
+- [New AWS and Microsoft regions](#aws-and-microsoft-regions)
+- [Microsoft social sign-on](#microsoft-social-sign-on)
+- [Flux updates](#flux-updates)
+- [InfluxDB community templates](#influxdb-community-templates)
+- [Load Data updates](#load-data-updates)
+- [Visualization updates](#visualization-updates)
+- [CLI updates](#cli-updates)
+- [API updates](#api-updates)
+- [Task updates](#task-updates)
+
+### AWS and Microsoft regions
+
+- Add support for the following new regions:
+
+  - [AWS US East (Virginia) region](/influxdb/cloud/reference/regions/#amazon-web-services-aws)
+  - [Microsoft Azure regions](/influxdb/cloud/reference/regions/#microsoft-azure)
+
+### Microsoft social sign-on
+
+- Add [Microsoft (Windows Live) social sign-on](https://cloud2.influxdata.com/signup). Use your Windows Live credentials to easily sign in to your InfluxDB Cloud account.
+
+### Flux updates
+
+- Update to [Flux v0.100.0](/influxdb/cloud/reference/release-notes/flux/#v0-100-0-2021-01-07).
+
+### InfluxDB community templates
+
+- Access any [InfluxDB community template](https://github.com/influxdata/community-templates#templates) directly in the Cloud user interface (UI). For more details, see how to [install and customize a template in the UI](/influxdb/cloud/influxdb-templates/cloud/).
+
+- Use the new [InfluxDB 2 Operational Monitoring community template](https://github.com/influxdata/community-templates/tree/master/influxdb2_operational_monitoring) to monitor InfluxDB 2.0.
+
+### Load Data updates
+
+  - Redesign the Load Data page to increase discovery and ease of use. Now, you can [load data from sources directly in the InfluxDB user interface](/influxdb/cloud/write-data/load-data/).
+  - Add support for new data sources:
+    - InfluxDB v2 Listener
+    - NSD
+    - OPC-UA
+    - Windows Event Log
+
+### Visualization updates
+
+  - Add new [Band Plot visualization](/influxdb/v2.0/visualize-data/visualization-types/band/).
+  - Usability improvements to visualizations:
+    - Add properties for storing your tick generation selections, including a `generateAxisTicks` property to turn this feature on and off.
```

Comment:
@bpalani, perhaps we should drop line 62 for now? 
@noramullen1 is currently updating documentation related to "Ticks" in visualizations